.github/workflows/tribits_lint.yml

```diff
@@ -0,0 +1,66 @@
+name: TriBITS Lint
+
+on:
+  pull_request:
+    branches: [ master ]
```

Comment:
@bartlettroscoe I originally had this as:

```yaml
on:
  schedule:
    - cron: '10  6  *  *  *' # Run 10 minutes after 2 AM EDT (6:10 UTC)
  push:
    branches: [ master ]
  pull_request:
    branches: [ master ]
```

just like `tribits_testing.yml`, but this caused GitHub Actions to not run my job. Any idea why that is?

Reply:
Note sure